bulk-upload-python-example/example_config.txt

```diff
@@ -1,5 +1,5 @@
 [domain]
 data-path = https://data.waylay.io
-resource-path = https://your-environment.waylay.io
+resource-path = https://staging.waylay.io
```

Comment:
I would not set `staging` as the default environment. This repo is public, so this could confuse people that they always have to use staging here.